web/src/beta/components/StorytellingPageSectionItem/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+type Props = {
+  icon: string;
+  title: string;
+  index: number;
+  active?: boolean;
+  onAction?: () => void;
+  onClick?: () => void;
+};
+
+const StorytellingPageSectionItem: FC<Props> = ({
+  icon,
+  title,
+  index,
+  active,
+  onAction,
+  onClick,
+}) => {
+  return (
+    <HorizontalBox>
+      <VerticalBox>
+        <Text size={"m"} color="#c7c5c5">
+          {index}
+        </Text>
+        <Icon icon={icon} />
+      </VerticalBox>
+      <TitleArea active={active}>
+        <Text onClick={onClick} size={"s"} color="#ffffff" otherProperties={{ cursor: "pointer" }}>
+          {title}
+        </Text>
+        <Icon icon="actionbutton" onClick={onAction} style={{ cursor: "pointer" }} />
```

Comment:
Does it work even if `cursor: pointer` is attached to `HotizontalBox`? I think we should set the pointer cursor as the entire style.